src/components/Checkbox/Checkbox.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "./Checkbox.css";
+
+function Checkbox({ title, checked, changeHandler }) {
+  return (
+    <label className="checkbox py-1">
+      <input type="checkbox" checked={checked} onChange={changeHandler} />{" "}
```

Comment:
remove {" "}  from code

Reply:
It's just to give space between checkbox and label. but I will try other way thanks